internal/csi/iam.go

```diff
@@ -77,15 +77,18 @@ type iamResource struct {
 	arn  *string
 }
 
-func createIAM() (*iam.IAM, error) {
-	sess, err := session.NewSession(&aws.Config{Region: aws.String("us-west-2")})
+func createIAM(awsRegion string) (*iam.IAM, error) {
```

Comment:
Should CreateIAMResources() just create the IAM client itself instead of this being a separate function?